Solve the grey screen when ios project is released.

Aaron009 opened the pull request.

brickpop commented:
Thank you for the PR, @Aaron009 

I understand that your code would be addressing the issue from https://github.com/brickpop/flutter-rust-ffi/issues/16 is this right?

I need to fully understand why these changes (mostly on the example app) would solve the release mode dropping the library symbols problem

Aaron009 commented:
I just modified the following under the file "ios/Classes/SwiftMylibPlugin.swift".

```diff
  public func dummyMethodToEnforceBundling() {
    // dummy calls to prevent tree shaking
+   rust_cstr_free(nil);
    rust_greeting("");
  }
```

Regarding the problem of gray screen, I deleted the project and created it again.

Regarding why the release mode deletes symbols, I think this may be an optimization of ios. Uncalled symbols are deleted to reduce the size of the package.
This is just my guess. If anyone knows it, please let me know.
